packages/augur-core/source/contracts/trading/AugurTrading.sol

```diff
@@ -47,7 +47,7 @@ contract AugurTrading is IAugurTrading {
     //  9:	tokensEscrowed
     event OrderEvent(address indexed universe, address indexed market, OrderEventType indexed eventType, uint8 orderType, bytes32 orderId, bytes32 tradeGroupId, address[] addressData, uint256[] uint256Data);
     event ProfitLossChanged(address indexed universe, address indexed market, address indexed account, uint256 outcome, int256 netPosition, uint256 avgPrice, int256 realizedProfit, int256 frozenFunds, int256 realizedCost, uint256 timestamp);
-    event MarketVolumeChanged(address indexed universe, address indexed market, uint256 volume, uint256[] outcomeVolumes);
+    event MarketVolumeChanged(address indexed universe, address indexed market, uint256 volume, uint256[] outcomeVolumes, uint256 timestamp);
```

Comment:
This is for dev-only?

Reply:
No it’s needed for real

Reply:
when not in fake-time, wouldn't the log event time be identical?

Reply:
what is log event time? There is no time associated with logs AFAIK, just a blockNumber. This can be used to look up a time but requires a request per block

Reply:
got it, so it's on-chain event emit to avoid off-chain block time lookups. i figured you already had all timestamp info for blocks for other non-event-based processing, but i haven't looked into the ingest system as much as i have the contracts.